src/univariate/locationscale.jl

```diff
@@ -1,129 +1,131 @@
 """
-    LocationScale(μ,σ,ρ)
+    AffineDistribution(μ, σ, ρ)
 
-A location-scale transformed distribution with location parameter `μ`,
-scale parameter `σ`, and given univariate distribution `ρ`.
+A shifted and scaled (affinely transformed) version of `ρ`.
 
-If ``Z`` is a random variable with distribution `ρ`, then the distribution of the random
-variable
+If `Z` is a random variable with distribution `ρ`, then `AffineDistribution(μ, σ, ρ)` is
+the distribution of the random variable
 ```math
-X = μ + σ Z
+X = μ + σ * Z
 ```
-is the location-scale transformed distribution with location parameter `μ` and scale
-parameter `σ`.
 
-If `ρ` is a discrete distribution, the probability mass function of
-the transformed distribution is given by
+If `ρ` is a discrete univariate distribution, the probability mass function of the 
+transformed distribution is given by
 ```math
 P(X = x) = P\\left(Z = \\frac{x-μ}{σ} \\right).
 ```
-If `ρ` is a continuous distribution, the probability density function of
-the transformed distribution is given by
+
+If `ρ` is a continuous univariate distribution with probability density function `f_Z`,
+the probability density function of the transformed distribution is given by
 ```math
-f(x) = \\frac{1}{σ} ρ \\! \\left( \\frac{x-μ}{σ} \\right).
+f_X(x) = \\frac{1}{|σ|} f_Z\\left( \\frac{x-μ}{σ} \\right).
 ```
 
+We recommend against using the `AffineDistribution` constructor directly. Instead, use 
+`+`, `-`, `*`, and `/`. These are optimized for specific distributions and will automatically
+fall back on `AffineDistribution` if they need to.
+
+Affine transformations of discrete variables are easily affected by rounding errors. If you
+are getting incorrect results, try using exact `Rational` types instead of floats.
+
 ```julia
-LocationScale(μ,σ,ρ) # location-scale transformed distribution
-params(d)            # Get the parameters, i.e. (μ, σ, and the base distribution ρ)
-location(d)          # Get the location parameter
-scale(d)             # Get the scale parameter
+d = σ * ρ + μ       # Create location-scale transformed distribution
+params(d)           # Get the parameters, i.e. (μ, σ, ρ)
 ```
-
-External links
-[Location-Scale family on Wikipedia](https://en.wikipedia.org/wiki/Location%E2%80%93scale_family)
 """
-struct LocationScale{T<:Real, S<:ValueSupport, D<:UnivariateDistribution{S}} <: UnivariateDistribution{S}
+struct AffineDistribution{T<:Real, S<:ValueSupport, D<:UnivariateDistribution{S}} <: UnivariateDistribution{S}
```

Comment:
We will want to allow different types for `mu` and `sigma`. If we don't export `AffineDistribution`, we are allowed to make these changes in one of the follow-up PRs though.

Reply:
I'll make it in the next one so we can get this one merged :+1: